pootle/templates/editor/units/xhr_tm.html

```diff
@@ -9,7 +9,9 @@
           dir="<%= store.source_dir %>"><%= PTL.editor.doDiff(unit.source[0], sugg.source) %></div>
         <div class="suggestion-translation" lang="<%= store.target_lang %>"
           dir="<%= store.target_dir %>"><%= PTL.utils.fancyHl(sugg.target) %></div>
-        <% if (sugg.project) { %>
+        <% if (!sugg.is_local) { %>
```

Comment:
TM is TM.  Overloading the user with information about external or internal is not helpful.  They likely already know this by the project name.  Lets have type of string that converys the information.  The only difference would be that external in our case would not know the contributors name (though I can imagine some formats could).